etc/services/preprints-inarxiv.json

```diff
@@ -0,0 +1,44 @@
+{
+  "@class" : "org.jasig.cas.services.RegexRegisteredService",
+  "id" : 203948234207257,
+  "name" : "INA-Rxiv",
+  "description" : "",
+  "serviceId" : "^https?://(localhost|127\\.0\\.0\\.1):5000/(login|logout)/?\\?next=https?(%3A|:)(%2F|/)(%2F|/)((local\\.nutrixiv\\.(org|com)(%3A|:)4200)|((localhost|127\\.0\\.0\\.1)(%3A|:)5000(%2F|/)preprints(%2F|/)inarxiv))($|%2F|/).*",
```

Comment:
ah I forgot to replace this one, I think this happens for service, 
need a hot fix